dhall-docs/src/Dhall/Docs.hs

```diff
@@ -81,6 +75,36 @@ parseOptions =
        <> Options.Applicative.metavar "OUTPUT"
        <> Options.Applicative.help "Directory where your docs will be generated"
        <> Options.Applicative.value "docs" )
+    <*> parsePackageNameResolver
+
+  where
+    parseStrategy :: Parser GenerationStrategy
+    parseStrategy =
+        Options.Applicative.flag
+            MultiPage
+            SinglePage
+            (   Options.Applicative.long "single-page"
+            <>  Options.Applicative.help
+                    (  "Generate a single page HTML documentation. By default,"
+                    <> "the tool will generate a multi-page documentation"
```

Comment:
This was previously multi-line string literals, but for some reason the first character from the second line was removed so I ended up concatenating several strings